src/acme/ui/CompanyUI.java

```diff
@@ -177,13 +175,15 @@ public void actionPerformed(ActionEvent e)
         @Override
         public void actionPerformed(ActionEvent e)
         {
-            // IMPLEMENT
-            
-
-//            System.out.println(field1.getText());
-//            Integer.parseInt(field2.getText());
-            
-            
+            getCompany().update();
```

Comment:
You need to get all of the fields from your UI and update the company with them before you call company.update